pkg/cmd/server/kubernetes/master/master_test.go

```diff
@@ -32,9 +25,8 @@ func TestNewMasterLeasesHasCorrectTTL(t *testing.T) {
 		t.Fatalf("error updating lease: %v", err)
 	}
 
-	etcdClient := server.Client
-	keys := client.NewKeysAPI(etcdClient)
-	resp, err := keys.Get(context.Background(), etcdtest.PathPrefix()+"/masterleases/1.2.3.4", nil)
+	etcdClient := server.V3Client
+	resp, err := etcdClient.Get(context.Background(), etcdtest.PathPrefix()+"/masterleases/1.2.3.4", nil)
```

Comment:
@smarterclayton 5 lines or so down.  etcd3 doesn't have TTL on each node.  Can you figure out what we need to do here?